src/renderer/entities/wallet/model/wallet-model.ts

```diff
@@ -8,6 +8,7 @@ import { modelUtils } from '../lib/model-utils';
 import { accountUtils } from '../lib/account-utils';
 
 const $wallets = createStore<Wallet[]>([]);
+console.log($wallets.getState());
```

Comment:
Remove
If you'd like to watch store value you should use `$store.watch((val) => ...);`